### ISSUE TITLE
Loading module that defines a grouping with a list with no keys and no 'config' statement leads to an error

### ISSUE THREAD

**Issue by panmanio** [CESNET/libyang#1826]
Hi,

having the following modules:

`mod_a.yang`:

```yang
module mod_a {
  namespace "x:example:mod_a";
  prefix "ma";

  import mod_b { prefix "mb"; }

  container root {
    config false;
    uses mb:listgroup;
  }
}
```

`mod_b.yang`:

```yang
module mod_b {
  namespace "x:example:mod_b";
  prefix "mb";

  grouping listgroup {
      list mylist {
          leaf name {
              type string;
          }
          leaf value {
              type uint32;
          }
      }
  }
}
```

We see that `listgroup` grouping defines a list that has no keys defined. Loading `mod_a` is successful and correct: we instantiate `mylist` as a state data tree. Loading `mod_b` afterwards is also successful:

```console
$ yanglint
> load mod_a
> load mod_b
>
```

However, reversing module loading order leads to an error:

```console
$ yanglint
> load mod_b
libyang[0]: Missing key in list representing configuration data. (path: /mod_b:{grouping='listgroup'}/mylist)
```

**Comment by michalvasko** [comment]
Please update to the latest `devel` branch, I believe it was fixed by 2d6507aeaa070284522bd3ae3b493142be0bf8dc.

**Comment by panmanio** [comment]
Thank you, commit 2d6507a helps indeed.
